samples/app-identity-linking-with-sso/nodejs/views/tab.ejs

```diff
@@ -127,12 +127,12 @@
 
                             if (userDetails.is_google_signed_in) {
                                 is_google_signed_in = userDetails.is_google_signed_in
-                                getGoogleDetails(userDetails.google_token);
+                                getGoogleDetails();
```

Comment:
We are not getting google details :)
We are getting the user details for a google account =>getGoogleAccountDetails();